experimental/language-generation/docs/lg-file-format.md

```diff
@@ -46,30 +83,29 @@ Here is an example that shows the simple IF ... ELSE conditional response templa
 - ELSE: 
     - good evening
 ```
-
 Here's another example that shows IF ... ELSEIF ... ELSE conditional response template definition. 
 
+Note that you can include references to other simple or conditional response templates in the variation for any of the conditions. 
 ```markdown
 # timeOfDayGreeting
 - IF: @{timeOfDay == 'morning'}
-    - good morning
-- ELSEIF: {timeOfDay == 'afternoon'}
-    - good afternoon
+    - @{morningTemplate()}
+- ELSEIF: @{timeOfDay == 'afternoon'}
+    - @{afternoonTemplate()}
 - ELSE: 
-    - good evening
+    - I love the evenings! Just saying. @{eveningTemplate()}
 ```
-
 ### Switch..Case
-Apart from IF ... ELSEIF ... ELSE construct, you can also use the SWITCH ... CASE ... DEFAULT construct. All conditions are expressed using the [Common expression language][3]. Condition expressions are enclosed in curly brackets - {}
+Apart from IF ... ELSEIF ... ELSE construct, you can also use the SWITCH ... CASE ... DEFAULT construct. All conditions are expressed using the [Common expression language][3]. Condition expressions are enclosed in curly brackets - @{}
 
```

Comment:
is the behavior of the switch evaluate from top to bottom and execute the first that match?

Reply:
Updated doc to clarify.